exercises/minesweeper/minesweeper.spec.js

```diff
@@ -0,0 +1,181 @@
+var Minesweeper = require('./minesweeper');
+
+describe('Minesweeper()', function() {
+
+  it('handles no rows', function() {
+    var minesweeper = new Minesweeper();
+    expect(minesweeper.annotate([])).toEqual([]);
+  });
+
+  xit('handles no columns', function() {
+    var minesweeper = new Minesweeper();
+    expect(minesweeper.annotate([""])).toEqual([""]);
+  });
+
+  xit('handles no mines', function() {
+    var minesweeper = new Minesweeper();
+    var input = [
+        "   ",
+        "   ",
+        "   "
+    ];
+    var expected = [
+        "   ",
+        "   ",
+        "   "
+      ];
+    expect(minesweeper.annotate(input)).toEqual(expected);
+  });
+
+  xit('handles board with only mines', function() {
+    var minesweeper = new Minesweeper();
+    var input = [
+        "***",
+        "***",
+        "***"
+    ];
+    var expected = [
+        "***",
+        "***",
+        "***"
+    ];
+    expect(minesweeper.annotate(input)).toEqual(expected);
+  });
+
+  xit('handles mine surrounded by spaces', function() {
+    var minesweeper = new Minesweeper();
+    var input = [
+        "   ",
+        " * ",
+        "   "
+    ];
+    var expected = [
+        "111",
+        "1*1",
+        "111"
+      ];
+    expect(minesweeper.annotate(input)).toEqual(expected);
+  });
+
+  xit('handles space surrounded by mines', function() {
+    var minesweeper = new Minesweeper();
+    var input = [
+        "***",
+        "* *",
+        "***"
+      ];
+    var expected = [
+        "***",
+        "*8*",
+        "***"
+      ];
+    expect(minesweeper.annotate(input)).toEqual(expected);
+  });
+
+  xit('handles space surrounded by mines', function() {
+    var minesweeper = new Minesweeper();
+    var input = [
+        "***",
+        "* *",
+        "***"
+      ];
+    var expected = [
+        "***",
+        "*8*",
+        "***"
+      ];
+    expect(minesweeper.annotate(input)).toEqual(expected);
+  });
+  xit('handles horizontal line', function() {

```

Comment:
A blank line here would be nice, and consistent with the rest of your lovely tests.